source/docs/appendices/migration-guides.md

```diff
@@ -9,6 +9,11 @@ Our goal is to make migrations as smooth as possible. This is why we try to make
 
 ## `2.5.0` -> `2.6.0`
 
+### Minimum Node.js version
+
+Node.js 10.x reaches the end of support in the end of April 2021. As a result,
```

Comment:
I suggest adding a direct link to the official documentation:
```suggestion
Node.js 10.x [reaches its end of life in the end of April 2021](https://nodejs.org/en/about/releases/). As a result,
```

Reply:
done in 907eaeb